journal-week-1.md

```diff
@@ -0,0 +1,16 @@
+# One Down Looking Forward for Eleven More 
+
+
+The first week was a shock, I indeed entered a boot camp. No hanky panky. Starting from the prep-work and the rest of the week, all of us from the class had the taste of brain squeezing experience. 
```

Comment:
"Hanky Panky" _could_ be seen as a loaded term - I can explain in lab if you like. I would not use it here though. :) If you need help brainstorming more phrases, come see me.